### DISCUSSION TITLE
Why don't schema-enabled box return BASIC `WWW-Authenticate` header ?

### DISCUSSION THREAD

**Discussion by yoh1496** [personium/personium-core#642] [Q&A]
In case I send HTTP GET request to box URL which schema URL is set, its response differs from one which schema URL is not set.

### Description

Examples are below.

#### HTTP GET to box which is not set schema.

```bash
curl https://user.pds.example.com/articles -i
HTTP/1.1 401
Date: Thu, 02 Sep 2021 05:50:39 GMT
Content-Type: application/json
Content-Length: 82
Connection: keep-alive
WWW-Authenticate: Bearer realm="https://user.pds.example.com/"
WWW-Authenticate: Basic realm="https://user.pds.example.com/"
X-Personium-Version: 1.7.22
Server: Personium
```

#### HTTP GET to box which is set shema.

```bash
$ curl https://user.pds.example.com/hogehoge -i
HTTP/1.1 401
Date: Thu, 02 Sep 2021 05:50:32 GMT
Content-Type: application/json
Content-Length: 82
Connection: keep-alive
WWW-Authenticate: Bearer realm="https://user.pds.example.com/"
X-Personium-Version: 1.7.22
Server: Personium
```

**Comment by shimono** [comment]
Schema-enabled box :  may require app (schema) authn for box contents access 　
  -> may not be able to let users authenticate themselves with their id/pw.
  -> Basic auth won't work.
  
 
Schema-disabled box :  cannot handle app (schema) authn  for box contents access.  
  -> can let user authenticate themselves with their id/pw.  
  -> Basic auth may work.
